objectivec/GPBMessage.m

```diff
@@ -2027,7 +2027,12 @@ - (void)parseMessageSet:(GPBCodedInputStream *)input
       [newInput release];
     } else {
       GPBUnknownFieldSet *unknownFields = GetOrMakeUnknownFields(self);
-      [unknownFields mergeMessageSetMessage:typeId data:rawBytes];
+      // rawBytes was created via a NoCopy, so it can be resusing a
```

Comment:
reusing

Reply:
Do we have a unit test for this?

Reply:
I didn't see a clean way to test this one without trying to mock out some dummy NSData and craft a flow to hit this case and validate what was called.